config.json

```diff
@@ -100,6 +100,17 @@
         "strings"
       ]
     },
+    {
+      "slug": "bank-account",
+      "uuid": "8beedf70-28b4-4cfb-ab53-309ee6f6aa78",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
Pick any exercise with the value `"core": true`. Something where you feel like this should be available after.